pages/index.tsx

```diff
@@ -144,77 +274,258 @@ export default function Home() {
         <meta name="viewport" content="width=device-width, initial-scale=1" />
         <meta name="viewport" content="width=device-width, initial-scale=1" />
       </Head>
-      <main>
-        <nav>
-          <Image alt="Prisma logo" src="/logo.svg" width={105} height={32} />
-          <span className="badge">Early Access</span>
+      <main className={styles.main}>
+        <div className={styles.nav}>
+          <a href="https://www.prisma.io/">
+            <Image alt="Prisma logo" src="/logo.svg" width={105} height={32} />
+          </a>
           <span style={{ flex: 1 }}></span>
           <a
-            href="https://www.prisma.io/data-platform/accelerate"
+            href="https://github.com/prisma/accelerate-speed-test"
             target="_blank"
+            className={styles.githubIcon}
             rel="noopener noreferrer"
           >
-            <Button variant="secondary">Join the waitlist</Button>
+            <img src={"./github.svg"} width="32px" height="32px" />
           </a>
-        </nav>
-        <header>
-          <Title titleProps={{ style: { fontSize: `2rem` } }}>
-            Accelerate Speed test
-          </Title>
-          <p>
-            The test will run for ~5 seconds. See how many requests Accelerate
-            can process sequentially over that time.
+          <WebsiteButton
+            href="https://www.prisma.io/data-platform/accelerate"
+            target="_blank"
+            rel="noopener noreferrer"
+            className={styles.waitlistbtn}
+            color="teal"
+            type="secondary"
+          >
+            Join the waitlist
+          </WebsiteButton>
+        </div>
+        <header className={styles.header}>
+          <p className={styles["eyebrow-headline"]}>
+            Accelerate
+            <span className="green-badge">Early Access</span>
           </p>
+          <h2 className={styles.h2}>Speed up your database queries with an automated global cache</h2>
+          <div>
+            <p>
+              This speed test runs a simple count query on a dataset with 500k rows and shows the results with and without the <a href="https://www.prisma.io/data-platform/accelerate">Accelerate</a> cache.
```

Comment:
```suggestion
              This speed test runs a simple count query on a dataset with 500k rows and shows the results with and without the <a href="https://www.prisma.io/data-platform/accelerate" target="_blank" rel="noopener noreferrer">Accelerate</a> cache.
```

I would open this in another tab like the rest of the links